File: C_Language_Batch0342/task02.c
#include<stdio.h>
void sum(int a,int b)
{
	int c;
	c = a,b; 
	printf("%d + %d = %d",a,b,c);
}
void main()
{
	int a,b; 
	printf("Enter 2 number : ");
	scanf("%d,%d",&a,&b); 

	sum(a,b); 
}

File: JavaCodersBatch5_8/Task/Task02.java

import java.util.*;
class Task02 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		System.out.print("Enter a number : ");
		int n = sc.nextInt();

		int temp=n;
		int sum=0,rem;

		while(n>0)
		{
			rem = n%10;
			sum+=rem;
			n=(int)n/10;
		}

		if(temp%sum==0)
		{
			System.out.println("Harshad number.");
		}
		else {
			System.out.println("Not Harshad number.");
		}
	}
}